app.js

```diff
@@ -122,6 +122,7 @@ class ChallengeCard {
     const modal = new Modal();
     bookButton.addEventListener("click", () => {
       modal.open(this.data.title, this.data.id);
+      updateParticipantsDisplay(this.data.id); //Shows the amount of participants in the select tag. 
```

Comment:
I understand why you've added a comment and I think it's a very good ambition to make the code easy to read by adding comments. The problem is that code can often become less easy to read, or even confusing, because of comments, so I generally recommend against having comments unless the code really needs it.

Here's what I wrote on the subject to a classmate of yours, who had moved some code but accidentally forgot to move the comment that documented the code, which may seem like a silly mistake but is actually extremely common:

> My theory is that most programmers stop reading comments, unless they don't understand the code. The comments are colored differently and programmers, being lazy, just filter them out to have to read less. This makes sense, but it does mean that when you make changes to code, it's very easy to forget updating comments to reflect those changes.
> 
> For that reason, it's usually a good idea to avoid having comments, and instead, writing the code in a way that makes it easily readable, using good variable names etc.

In this case, I think the name of the comment already communicates almost the exact same thing as the comment, rendering the comment redundant. If you feel that the name is not good enough, maybe you could pick a name that you feel better conveys what the function does. In this case, it could even be `showParticipantCountInSelect()`, which is almost exactly what you wrote in your comment.